pkg/cvo/sync_worker.go

```diff
@@ -274,7 +281,7 @@ func (w *SyncWorker) syncPayload(ctx context.Context, work *SyncWork,
 				Release:            desired,
 				LastTransitionTime: time.Now(),
 			})
-			return nil, err
+			return implicitlyEnabledCaps, err
```

Comment:
nit: we don't need these or the other `err != nil` touches, because:

* [`loadUpdatedPayload`][1] is going to throw them out and return a `nil` cap set on errors, and
* [`Update`][2] doesn't care what the returned cap set is when there's an error.

We only care about the returned cap set when the error is `nil`.

[1]: https://github.com/openshift/cluster-version-operator/blob/4e074889fc30cef47dae8b1309648be821bd02d8/pkg/cvo/sync_worker.go#L377
[2]: https://github.com/openshift/cluster-version-operator/blob/4e074889fc30cef47dae8b1309648be821bd02d8/pkg/cvo/sync_worker.go#L458